cabal.project

```diff
@@ -234,8 +234,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/plutus
-  tag: 826c2514a40e962c2e4d56ce912803a434cc28fe
-  --sha256: 04jryrgi2wggkf4a994smbcp8j39clshbzfnvfvvk7lq0sm8kssz
+  tag: 985bd37845cd0996fc38eaccd1db1a9ad3edab3a
```

Comment:
3091 of `cls` uses `95e5d7488451e43062ca84d5376b3adcc465f1cd` for `plutus` -- should we match that here?

Reply:
Erm... it uses the same version: https://github.com/input-output-hk/cardano-ledger-specs/blob/309178369eebe9ce02171987f6d23c7778f7db26/cabal.project#L72

Reply:
Aha! I glanced too quickly. `95e5d7488451e43062ca84d5376b3adcc465f1cd` is for `https://github.com/Quid2/flat.git`, not for `plutus`.